syntaxes/template.json

```diff
@@ -26,6 +29,33 @@
           "include": "source.js"
         }
       ]
+    },
+
+    "propertyBinding": {
+      "begin": "(\\[.*\\])(=)(\"|')",
+      "beginCaptures": {
+        "1": {
+          "name": "entity.other.attribute-name.html"
+        },
+        "2": {
+          "name": "punctuation.separator.key-value.html"
+        },
+        "3": {
+          "name": "punctuation.definition.string.begin.html string.quoted.html"
+        }
+      },
+      "end": "\\3",
```

Comment:
Does this refer to the 3rd capture group above, or is there special meaning to the literal value 3?

Reply:
Yes, this references the 3rd capture group (so that we can grab the correct closing quote type)